include/openssl/ssl.h

```diff
@@ -2058,6 +2058,8 @@ __owur const struct openssl_ssl_test_functions *SSL_test_functions(void);
 
 extern const char SSL_version_str[];
 
+void DTLS_set_timer_cb(SSL *s, unsigned (*cb)(SSL *s, unsigned timer));
```

Comment:
Please moce and use here your `typedef `on callback function : 
`typedef unsigned (*DTLS_timer_cb)(SSL *s, unsigned timer) ;`
It will improve the readability.